Clarify out-of-scope (#50)
tfpauly/draft-happy-eyeballs-v3: charter.md

to report such cases that might otherwise be hidden due to automatic switching
from one technology to another.

The working group will focus on a connection establishment algorithm for a

> client that starts with a fully-qualified domain name (FQDN) and wants to end
> up with a single established connection to a single server IP address on a

Reviewer: ```suggestion
client that starts with a fully-qualified domain name (FQDN) and wants to
obtain a single established connection to a single server IP address on a
```
No strong opinion, but "end up with" just sounded a bit weird.

Reviewer 2: +1. I also don't love the personification (the use of the word "wants"). How about something like:

```
The working group will focus on a connection establishment algorithm that
runs on clients. The algorithm takes as input a fully-qualified domain name
(FQDN), and outputs a single established connection to a single server IP
address on a single network.
```

Author: Cool, took this suggestion with one minor edit!

> The working group will focus on a connection establishment algorithm for a
> client that starts with a fully-qualified domain name (FQDN) and wants to end
> up with a single established connection to a single server IP address on a
> single network. While the algorithm could apply to scenarios with multiple

Author: ```suggestion
The working group will focus on a connection establishment algorithm that
runs on clients. The algorithm takes as input a fully-qualified domain name
(FQDN), and results in a single established connection to a single server IP
address on a single network. While the algorithm could apply to scenarios with multiple
```

> networks to choose between or use simultaneously, or deal with pools of

Reviewer: ```suggestion
networks to choose between or to use simultaneously, or could deal with pools of
```
Not totally sure, but I think the added words add clarity to the "or"s.

> multiple connections, such expansions are out of scope for the working group

Reviewer: I think "expansions" is fine, but wondering if "circumstances" or "situations" or some other word is better?

Author: Maybe scenarios:
```suggestion
multiple connections, such scenarios are out of scope for the working group
```

> address on a single network. While the algorithm could apply to scenarios with multiple
> networks to choose between or to use simultaneously, or could deal with pools of
> multiple connections, such scenarios are out of scope for the working group
> deliverables.
> 

Author: ```suggestion
address on a single network. While the algorithm could apply to scenarios with
multiple networks to choose between or to use simultaneously, or could deal
with pools of multiple connections, such scenarios are out of scope for the
working group deliverables.

```
